examples/parallel_bert.py

```diff
@@ -0,0 +1,192 @@
+import os
+import argparse
+import functools
+import pandas as pd
+import pathlib
```

Comment:
you should not import pathilib. Just import what you need. For example 
`from pathlib import Path`

Reply:
It is just a good practice :-) thanks